### PR TITLE
Create German translation

### PR DESCRIPTION
**Pull request by reckel-jm**
I've created a file with the German translation.

### CODE REVIEW

**Review by vrybant — APPROVED** [review]

**Comment by reckel-jm** [comment]
Hey @vrybant, can you merge this pull request after you have approved it? You have approved it, but it remains unmerged. God bless you and thanks!

**Comment by vrybant** [comment]
Hi @reckel-jm, thank you for the German translation!

**Comment by reckel-jm** [comment]
@vrybant you are welcome!

**Pull request merged**
